docs/src/frontend/index.md

```diff
@@ -696,6 +696,25 @@ acc.tokensAccepted() => Promise<Array<Token>>
 Returns a Promise for an array of tokens that are accepted by `acc`.
 On networks which do not keep track of this information (e.g. Ethereum), this returns an empty array.
 
+---
+@{ref("js", "appOptIn")}
+```js
+ctc.appOptIn() => Promise<void>
+```
+
+This does nothing on some networks.
+On others, it opts into the contract `ctc`.
+
+---
+@{ref("js", "appOptedIn")}@{ref("js", "stdlib.accountAppOptedIn")}
+```js
+acc.appOptedIn(contract) => Promise<boolean>
+stdlib.accountAppOptedIn(account, contract) => Promise<boolean>
```

Comment:
The naming convention for things like this is to use the same name with no prefix, so `stdlib.appOptedIn` would be the name